src/Auth0Client.ts

```diff
@@ -497,13 +500,24 @@ export default class Auth0Client {
     } else {
       delete options.client_id;
     }
-    ClientStorage.remove('auth0.is.authenticated');
+
     const { federated, localOnly, ...logoutOptions } = options;
+
+    if (localOnly && federated) {
+      throw new Error(
+        `Logging out of identity providers using the 'federated' option will not work when specifying a local logout using 'localOnly'`
```

Comment:
I wonder if we need to be more direct here in our messaging. We could just say "It is invalid to specify both `federated` and `localLogout` at the same time" or something to that effect. Saying "will not work" but then not explaining _why_ it doesn't work will just invite more questions.

Reply:
Also, as this string doesn't use any string interpolation, let's change the back-ticks here for regular single-quotes, just for style consistency.